### PR TITLE
Added community meeting information in the documentation #718

### PR DESCRIPTION
**Pull request by Sanskarzz**

### CODE REVIEW

**Review by shubham-cmyk — CHANGES_REQUESTED** [review]
Can you add the Zoom Meeting Link here

**Comment by Sanskarzz** [comment]
it is provided inside agenda and meeting notes link

**Review by shubham-cmyk — COMMENTED** [review]
Over here

File: website/docs/community.md [hunk 1]
```diff
@@ -18,6 +18,14 @@ Chainsaw is built and maintained under the Kyverno umbrella but decisions are
 
 Join our slack channel [#kyverno-chainsaw](https://kubernetes.slack.com/archives/C067LUFL43U) to meet with users, contributors and maintainers.
 
+## Community Meetings
+
+This is a public, weekly for Kyverno-Chainsaw maintainers to make announcements and provide project updates, and request input and feedback. This forum allows community members to raise agenda items of any sort, including but not limited to any PRs or issues on which they are working.
+
+Weekly every Thursday at 2:00 PM UTC
```
Comment (shubham-cmyk) [inline website/docs/community.md:25]: over here

**Comment by eddycharly** [comment]
> it is provided inside agenda and meeting notes link

I need to update that !

**Comment by eddycharly** [comment]
@Sanskarzz can you sign-off your commits ?

File: website/docs/community.md [hunk 2]
```diff
@@ -18,6 +18,20 @@ Chainsaw is built and maintained under the Kyverno umbrella but decisions are
...
+<<<<<<< HEAD
+=======
+
+[Zoom Meeting](https://zoom.us/j/99815137900)
+
+>>>>>>> 1616007 (Added Zoom Meeting link)
```
Comment (eddycharly) [inline website/docs/community.md:31]:
```suggestion
[Zoom Meeting](https://zoom.us/j/99815137900)
```

**Review by eddycharly — APPROVED** [review]

**Pull request merged**